pyartcd/pyartcd/pipelines/gen_assembly.py

```diff
@@ -91,7 +91,7 @@ async def run(self):
             pr = await self._create_or_update_pull_request(assembly_definition)
 
             # Sends a slack message
-            message = f"Hi @release-artists , please review assembly definition for {self.assembly}: {pr.html_url}"
+            message = f"Hi @release-artists , please review assembly definition for {self.assembly}: {pr.html_url}\n\nthe nightlies are {candidate_nightlies}\n\nthe inflight release is {self.in_flight}"
```

Comment:
ehh why include nightlies? I would say let's not make the message more noisy, inflight is good